src/site/content/en/progressive-web-apps/codelab-make-installable/index.md

```diff
@@ -107,5 +108,5 @@ Congratulations, you app is now installable!
 
 Here are some additional things that you can do:
 
-+  [Detecting if your app is launched from the home screen](https://developers.google.com/web/fundamentals/app-install-banners/#detect-mode)
++  [Detecting if your app is launched from the home screen](/customize-install/#detect-mode)
 +  [How to show the native app install prompt instead](https://developers.google.com/web/fundamentals/app-install-banners/native)
```

Comment:
```suggestion
+  [Detect if your app is launched from the home screen](/customize-install/#detect-mode)
+  [Show the native app install prompt instead](https://developers.google.com/web/fundamentals/app-install-banners/native)
```

Reply:
The best practice for lists is to use parallel grammatical structures.

Reply:
Fixed.